### ISSUE TITLE
feature request: /discord embed code generation

### ISSUE THREAD

**Issue by aphkyle** [gustavwilliam/branding-bot#14]
will probably make a follow-up pr if this gets accepted, when im free soon

**Comment by gustavwilliam** [comment]
Could you explain what this feature would do? (You can edit the original message)

**Comment by aphkyle** [comment]
this feature should generate code if you're trying to make a discord bot, like when you dump that code in you'll get the identical embed.

**Comment by gustavwilliam** [comment]
That would be awesome!! Should I assign this issue to you @aphkyle?

**Comment by aphkyle** [comment]
> That would be awesome!! Should I assign this issue to you @aphkyle?

yes probably, but im busy for a while, ill prob pr after july or smth
